opendebates/static/js/registration.js

```diff
@@ -1,14 +1,19 @@
 (function() {
   $(window).load(function() {
     $(".form-group").each(function() {
-      if ($(this).find("label").length === 1) {
-        var text = $(this).find("label").hide().text();
+      var $label = $(this).find("label");
+      if ($label.length === 1) {
+        var text = $label.text();
```

Comment:
Yup, that's right.  Tooltips were previously completely hidden, because we didn't want field labels, just placeholders.  But we need a tooltip to explain why we're asking for a phone number so that we don't freak people out.  So this PR continues to take any text content from the label and put it in a placeholder instead, but retains the tooltip, and positions the tooltip "inside" the form field with some css positioning. 

Basically it turns

```
<label>Foo! Bar! Baz! <span>Morx</span></label>
<input>
```

into
```
<label><span>Morx</span></label>
<input placeholder="Foo! Bar! Baz!">
```